modules/use-default-security-policies.adoc

```diff
@@ -12,7 +12,9 @@ To view the default policies:
 
 * In the {product-title-short} portal, go to *Platform Configuration* -> *Policy Management*.
 
-The *Policies* view lists the default policies and includes the following parameters for each policy:
+The *Policies* view lists the default policies and enables you to configure the policies. 
```

Comment:
```suggestion
The *Policies* view also enables you to configure the policies. 
```
because we said right above there "To view the default policies"
don't need to say it again